python/ray/tune/tests/test_resource_updater.py

```diff
@@ -0,0 +1,41 @@
+import ray
+from ray.tests.conftest import *  # noqa
+from ray.tune.utils.resource_updater import ResourceUpdater
+
+
+def test_resource_updater(shutdown_only):
```

Comment:
Do you think it would be possible to structure the test in such a way that ray.init is called once but we can dynamically add/remove nodes and assert certain number of cpus/gpus?

Something like `test_multi_node_2.py` using a cluster fixture?

The worry I have about multiple ray.init() is what we are testing doesn't really match how this class is supposed to be used.

Reply:
I modified the test with cluster